astroquery/vo/examples/Registry.ipynb

```diff
@@ -0,0 +1,295 @@
+{
```

Comment:
Could you please remove this notebook (and/or rework the content into normal `rst` files? We don't have docs in astroquery in notebooks, and for tutorials, they are in another repo, https://github.com/astropy/astropy-tutorials.

Reply:
Yes, good point.  Sorry about that.  Just getting back to this PR now, and hope to get it into decent shape soon.

Reply:
no worries and no rush. I'm not sure about Adam's plans, but I won't be really available in the upcoming weeks to do any detailed review anyway.